content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -3464,6 +3460,22 @@ No response payload (`200 OK`).
 
 </CodeSample>
 
+<CodeResponse>
+
+```json
+{
+  "request_id": "<request_id>", // optional
+  "action": "mark_events_as_seen",
+  "type": "response",
+  "success": true,
+  "payload": {
+	//no response payload
```

Comment:
Same as above

Reply:
Comments removed in 6db52b2 - sorry it took so long